src/test/regress/expected/multi_distributed_transaction_id.out

```diff
@@ -142,6 +142,7 @@ INSERT INTO parallel_id_test VALUES (1234567), (1234567), (1234568), (1234568);
 ANALYSE parallel_id_test;
 SET LOCAL max_parallel_workers_per_gather TO 2;
 SET LOCAL parallel_tuple_cost TO 0;
+SET LOCAL parallel_setup_cost TO 0;
```

Comment:
here parallelism is essential, because we're testing that parallel workers have the same distributed transaction ID as the leader.